### PR TITLE
Create Rulesets for issue #18059 (Whoer.net.xml & Surfshark.com.xml)

### PR DESCRIPTION
**Pull request by bfosterscripps**
### Related PRs

- https://github.com/EFForg/https-everywhere/pull/18713
- https://github.com/EFForg/https-everywhere/pull/19550
- https://github.com/EFForg/https-everywhere/pull/19551

### Related Issues

- https://github.com/EFForg/https-everywhere/issues/18059

### CODE REVIEW

**Comment by bfosterscripps** [comment]
I believe this is ready for review @pipboy96

File: src/chrome/content/rules/Surfshark.com.xml [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+	Connection refused:
+		mor-ord.prod.surfshark.com (r)
+		mordor-orodruin-v001.prod.surfshark.com (r)
+		
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
```
Comment (pipboy96) [inline src/chrome/content/rules/Surfshark.com.xml:52]:
```suggestion
	r: connection refused
	t: timeout on https
```

File: src/chrome/content/rules/Surfshark.com.xml [hunk 2]
```diff
@@ -0,0 +1,189 @@
+<!--
+Note to ruleset creator: if you are aware of any nonfunctional hosts relevant
+to this ruleset, please list them below, replacing the example.
+
+Alternatively, if you are unaware of any such hosts, please delete this comment
+in its entirety.
+
+	Nonfunctional hosts in *.surfshark.com:
+		# Example: foobar.surfshark.com (m)
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Surfshark.com.xml:10]:
```suggestion
```

File: src/chrome/content/rules/Whoer.net.xml [hunk 3]
```diff
@@ -0,0 +1,85 @@
+<!--
+Note to ruleset creator: if you are aware of any nonfunctional hosts relevant
+to this ruleset, please list them below, replacing the example.
+
+Alternatively, if you are unaware of any such hosts, please delete this comment
+in its entirety.
+
+	Nonfunctional hosts in *.whoer.net:
+		# Example: foobar.whoer.net (m)
+
```
Comment (pipboy96) [inline src/chrome/content/rules/Whoer.net.xml:10]:
```suggestion
```

File: src/chrome/content/rules/Whoer.net.xml [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+		nl5-speed.whoer.net
+
+	Failed to connect, No route to host:
+		us5-speed.whoer.net
+
+	h: http redirect
+	m: certificate mismatch
+	r: connection refused
+	s: self-signed certificate
+	t: timeout on https
```
Comment (pipboy96) [inline src/chrome/content/rules/Whoer.net.xml:75]:
```suggestion
	r: connection refused
```

**Review by pipboy96 — CHANGES_REQUESTED** [review]

**Review by zoracon — APPROVED** [review]

**Pull request merged**